flixel/graphics/FlxAsepriteUtil.hx

```diff
@@ -0,0 +1,194 @@
+package flixel.graphics;
+
+import flixel.graphics.frames.FlxAtlasFrames;
+import flixel.system.FlxAssets;
+
+class FlxAsepriteUtil
+{
+	public static function loadAseAtlas(sprite:FlxSprite, graphic, data:FlxAsepriteJsonAsset, tagSuffix:String = ":")
+	{
+		final aseData = data.getData();
+		sprite.frames = FlxAtlasFrames.fromAseprite(graphic, aseData);
+		
+		for (frameTag in aseData.meta.frameTags)
+			sprite.animation.addByPrefix(frameTag.name, frameTag.name + tagSuffix);
+	}
+}
+
+typedef AtlasBase<T> = { frames:T }
+
+typedef AseAtlasBase<T> = AtlasBase<T> &
+{
+	var meta:AseAtlasMeta;
+}
+
+typedef AseAtlasArray = AseAtlasBase<Array<AseAtlasFrame>>;
+typedef TexturePackerAtlasArray = AtlasBase<Array<AseAtlasFrame>>;
+
+abstract Hash<T>(Dynamic)
+{
+	public inline function keyValueIterator():KeyValueIterator<String, T>
+	{
+		var keys = Reflect.fields(this).iterator();
+		return
+		{
+			hasNext: keys.hasNext,
+			next: ()->
+			{
+				final key = keys.next();
+				return { key:key, value:Reflect.field(this, key)};
+			}
+		};
+	}
+}
+
+typedef AseAtlasHash = AseAtlasBase<Hash<AseAtlasFrame>>;
+typedef TexturePackerAtlasHash = AtlasBase<Hash<TexturePackerAtlasFrame>>;
+
+typedef HashOrArray<T> = flixel.util.typeLimit.OneOfTwo<Hash<T>, Array<T>>;
+
+typedef AseAtlas = AseAtlasBase<HashOrArray<AseAtlasFrame>>;
+typedef TexturePackerAtlas = AtlasBase<HashOrArray<TexturePackerAtlasFrame>>;
+
+typedef AseAtlasMeta =
+{
+	var app:String;
+	var version:String;
+	var image:String;
+	var format:String;
+	var size:AtlasSize;
+	var scale:String;
+	var frameTags:Array<AseAtlasTag>;
+	var slices:Array<AseAtlasSlice>;
+	var layers:Array<AseAtlasLayer>;
+}
+
+typedef AseAtlasTag = { name: String, from:Int, to:Int, direction:String };
+
+typedef AseAtlasSlice =
+{
+	var name:String;
+	var color:String;
+	
+	/**
+	 * Info of at what frames the slice changes size
```

Comment:
Minor nit:

`Info of at what frames the slice changes size/position`

Since you can move them around and resize them per frame